src/utils/preprocess.ts

```
@@ -40,15 +41,22 @@
+  videoCaption?: string,
+  presentationStyle?: MulmoPresentationStyle,
+) => {
   const mulmoScript = _mulmoScript.__test_invalid__ ? _mulmoScript : mulmoScriptSchema.parse(_mulmoScript); // validate and insert default value
 
   const studio: MulmoStudio = rebuildStudio(currentStudio, mulmoScript, fileName);
 
   // TODO: Move this code out of this function later
   // Addition cloing credit
   if (mulmoScript.$mulmocast.credit === "closing") {
-    mulmoScript.beats.push(mulmoCredit(mulmoScript.beats[0].speaker)); // First speaker
+    const defaultSpeaker = MulmoPresentationStyleMethods.getDefaultSpeaker(presentationStyle ?? studio.script)
```

ESLint

[error] Insert `;`

(prettier/prettier)